.evergreen/find-python3.sh

```diff
@@ -150,7 +150,7 @@ is_virtualenv_capable() (
     # shellcheck source=/dev/null
     . "$tmp/bin/activate"
   elif [[ -f "$tmp/Scripts/activate" ]]; then
-    dos2unix "$tmp/Scripts/activate" || return
```

Comment:
@eramongodb this command was erroring out locally on @DmitryLukyanov's Windows machine.  I don't think it is strictly necessary, do you disagree?

Reply:
What was the nature of the error (missing `dos2unix`? an error during conversion?)?

This command is inherited from [older venv creation/activation scripts](https://github.com/mongodb-labs/drivers-evergreen-tools/blame/0638b75db86aae603fa882271a251e86309fe050/.evergreen/csfle/activate_venv.sh#L21) to address a [still-unresolved venv issue in Cygwin environments](https://bugs.python.org/issue32451) (context documented in [venv-utils.sh](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/9eff0be763b12b4417ca035dd18eb5170c9e4336/.evergreen/venv-utils.sh#L72)) where the activation script fails during execution/sourcing due to:
```
line 3: $'\r': command not found
```

This issue is still relevant on [many Windows Evergreen distros](https://spruce.mongodb.com/version/63f6668357e85a38474bdf1a/tasks) despite the introduction of "prefer newer Python versions" and virtualenv fallback behavior (note: despite successful virtual environment creation, logs reveal scripts are forced to select old Python binaries + fallback to virtualenv despite availability of newer Python binaries + venv, which also affects availability/selection of packages). Unfortunately, although this issue was [resolved by virtualenv in 20.0.33](https://github.com/pypa/virtualenv/issues/1818), it remains unresolved for venv, thus the manual `dos2unix` workaround was still deliberately included in the new creation/activation scripts to enable greater accessibility and use of venv across Evergreen distros.

We could consider demoting the error handling from `|| return` to `|| true` if a benign conversion error is preventing proper script execution in local environments, but given these scripts are designed to be executed in Evergreen environments, I suggest simply commenting it out locally as a quick workaround if the issue is also exclusive to the local machine.

Reply:
I believe the command was present but failing in this instance.  In the case of OIDC, there is cause to run these scripts quite often locally while writing the tests.  I'd be in favor of using `|| true`.

Reply:
Created https://github.com/mongodb-labs/drivers-evergreen-tools/pull/270 in response to this issue.